From ae69cf694578c3217a5524c957c37117c514a149 Mon Sep 17 00:00:00 2001
From: Christian Lamparter <chunkeey@googlemail.com>
Date: Sat, 6 Jul 2013 21:38:39 +0200
Subject: [PATCH 66/66] r92su: disable event tracing for out-of-tree module build

Signed-off-by: Christian Lamparter <chunkeey@googlemail.com>
---
 Makefile      | 4 ++--
 r92su/trace.h | 2 --
 2 files changed, 2 insertions(+), 4 deletions(-)

#diff --git a/Makefile b/Makefile
#index e45fc7c..543d05a 100644
#--- a/Makefile
#+++ b/Makefile
#@@ -5,10 +5,10 @@ KMOD_SRC ?= $(PWD)/r92su
# 
# # Each configuration option enables a list of files.
# 
#-KMOD_OPTIONS += CONFIG_R92SU=m CONFIG_R92SU_DEBUGFS=y CONFIG_R92SU_WPC=y CONFIG_R92SU_TRACING=y
#+KMOD_OPTIONS += CONFIG_R92SU=m CONFIG_R92SU_DEBUGFS=y CONFIG_R92SU_WPC=y
# 
# # Don't build any of the other drivers
#-EXTRA_CFLAGS += -DDEBUG -DCONFIG_R92SU=m -DCONFIG_R92SU_DEBUGFS=y -DCONFIG_R92SU_WPC=y -DCONFIG_R92SU_TRACING=y
#+EXTRA_CFLAGS += -DDEBUG -DCONFIG_R92SU=m -DCONFIG_R92SU_DEBUGFS=y -DCONFIG_R92SU_WPC=y
# 
# all:
# 	$(MAKE) -C $(KSRC) M=$(KMOD_SRC) $(KMOD_OPTIONS) $(MAKECMDGOALS) EXTRA_CFLAGS="$(EXTRA_CFLAGS)"
--- a/drivers/net/wireless/r92su/trace.h
+++ b/drivers/net/wireless/r92su/trace.h
@@ -262,11 +262,9 @@ DEFINE_EVENT(r92su_msg_event, r92su_dbg,
 
 #endif /* _R92SU_TRACE_H || TRACE_HEADER_MULTI_READ */
 
-/*
 #undef TRACE_INCLUDE_PATH
 #define TRACE_INCLUDE_PATH .
 #undef TRACE_INCLUDE_FILE
 #define TRACE_INCLUDE_FILE trace
 
 #include <trace/define_trace.h>
-*/
